retrieval/series_cache.go

```diff
@@ -393,12 +393,15 @@ func (c *seriesCache) refresh(ctx context.Context, ref uint64) error {
 	if metadata == nil {
 		// The full name didn't turn anything up. Check again in case it's a summary or histogram without
```

Comment:
Update the comment here, as checking against suffix is no longer only for summary or histogram - now it's for counter as well.

Reply:
Done.